### PR TITLE
[FIX] website_payment: missing translations in controller

### PR DESCRIPTION
**Pull request by Yenthe666**
Description of the issue/feature this PR addresses: Without this the error message will always be in English, no matter the language of the end-user.
This PR fixes this.

Current behavior before PR: The warning is always in English

Desired behavior after PR is merged: The warning is in the user his language, if there is a translation.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by Yenthe666** [comment]
@mart-e as discussed yesterday (https://github.com/odoo/odoo/blob/10.0/addons/website_payment/controllers/main.py#L81-L82)
I'm not completely sure if this exactly like you want it though. Can you double check please?

File: addons/website_payment/controllers/main.py [hunk 1]
```diff
@@ -78,8 +78,8 @@ def confirm(self, **kw):
         tx_id = request.session.pop('website_payment_tx_id', False)
         if tx_id:
             tx = request.env['payment.transaction'].browse(tx_id)
-            status = (tx.state == 'done' and 'success') or 'danger'
-            message = (tx.state == 'done' and 'Your payment was successful! It may take some time to be validated on our end.') or 'OOps! There was a problem with your payment.'
+            status = (tx.state == _('done') and _('success')) or _('danger')
```
Comment (xmo-odoo) [inline addons/website_payment/controllers/main.py]: tx.state is a selection field, so these are selection values and should not be translated.

Comment (Yenthe666) [inline addons/website_payment/controllers/main.py]: Hmm yep I was figuring as much. So I should remove the `_()` for `done`, `success` and `danger` right?

Comment (tivisse) [inline addons/website_payment/controllers/main.py]: Yes.

**Comment by Yenthe666** [comment]
@tivisse removed the `_()` and the `.pot` source strings that where no longer relevant. Can you recheck?

**Pull request merged**